mycity/mycity/intents/user_address_intent.py

```diff
@@ -0,0 +1,109 @@
+"""
+Functions for setting and getting the current user address
+"""
+
+from . import intent_constants
+
+
+def set_address_in_session(mcd):
+    """
+    Adds an address from the set address intent to the provided session object
+
+    :param mcd: MyCityRequestModel object
+    """
+    print(
+        '[module: user_address_intent]',
+        '[method: set_address_in_session]',
+        'MyCityDataModel received:',
+        str(mcd)
+    )
+    if 'Address' in mcd.intent_variables:
+        mcd.session_attributes[intent_constants.CURRENT_ADDRESS_KEY] = \
+            mcd.intent_variables['Address']['value']
+
+
+def create_set_address_intent_response(mcd):
+    """
+    Stores the user's address in the current session

```

Comment:
This comment isn't accurate

Reply:
Updated this. Note that the function comments overall need an update for consistency and accuracy.